src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
then it's a matter how you execute php-cs-fixer from grumphp.

that's `--path-mode=override`, take everything passed to cli command and ignore finder from config file. yet, it will ignore whole finder, including exclusions. you cannot partially honour finder and partially not.